lib/github/api.nit

```diff
@@ -466,7 +466,7 @@ class GithubAPI
 	#     assert repo != null
 	#     var comment = api.load_commit_comment(repo, 8982707)
 	#     assert comment.user.login == "Morriar"
-	#     assert comment.body == "For testing purposes..."
+	#     assert comment.body == "For testing purposes...\n"
```

Comment:
I'm puzzled by this change, was the test already broken or did the `int_fast64_t` modification trigger it to fail? If so, I'd like to check how, it does not seem normal to me.